targets/x86-xen_domu/config

```diff
@@ -0,0 +1,2 @@
+CONFIG_TARGET_x86=y
+CONFIG_TARGET_x86_xen_domu=y
```

Comment:
The file `config` can be deleted, the target and subtarget options are set implicitly by Gluon for some time.